src/main/resources/application-prod.properties

```diff
@@ -1,3 +1,5 @@
-feature.googleMapsEnabled=true
+feature.googleMaps.enabled=true
 google.maps.api-key=${GOOGLE_MAPS_API_KEY}
-google.maps.uri=https://maps.googleapis.com/maps/api/distancematrix/json
\ No newline at end of file
+google.maps.uri=https://maps.googleapis.com/maps/api/distancematrix/json
+spring.data.redis.host=localhost
```

Comment:
jeśli mamy te właściwości w application.properties to możesz je ominąć w "-prod" oraz "-local", a przynajmniej ich wartości. Czyli albo zostaw klucz albo usuń całkowicie, bo im więcej takich wartości nie zmieniających postaci rzeczy (bo i tak to jest nadpisywane dokładnie tą samą wartością) tym ciężej później to rozczytać